opensrp-reveal/src/test/java/org/smartregister/reveal/fragment/StatsFragmentTest.java

```diff
@@ -0,0 +1,101 @@
+package org.smartregister.reveal.fragment;
+
+import android.content.Context;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.TextView;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.powermock.reflect.Whitebox;
+import org.robolectric.Robolectric;
+import org.robolectric.RuntimeEnvironment;
+import org.smartregister.reveal.BaseUnitTest;
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.presenter.StatsFragmentPresenter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.smartregister.reveal.util.Constants.SyncInfo.INVALID_CLIENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.INVALID_EVENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.SYNCED_CLIENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.SYNCED_EVENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.TASK_UNPROCESSED_EVENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.UNSYNCED_CLIENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.UNSYNCED_EVENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.VALID_CLIENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.VALID_EVENTS;
+
+/**
+ * Created by Richard Kareko on 2/25/20.
+ */
+
+public class StatsFragmentTest extends BaseUnitTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+
+    @Mock
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5066421)